tests/test_platform.py

```diff
@@ -35,6 +35,14 @@ def test_create_platform_error():
         _ = create_platform("nonexistent")
 
 
+def test_create_platform_from_hardware():
+    original_value = os.environ.get(PLATFORMS)
+    os.environ[PLATFORMS] = str(Path(__file__).parent)
+    _ = create_platform("dummy_hardware")
```

Comment:
Maybe we could assert something about it.
E.g. `isinstance(..., Platform)` is already a basic property which should be always preserved. Maybe even having more than 0 qubits, or things like that.

Reply:
Done in fa1b22e26d91b4d341220c3a71c5c3020d87128b.